src/index.ts

```diff
@@ -265,6 +266,23 @@ class FSMerger {
     result.sort((entryA, entryB) => (entryA.relativePath > entryB.relativePath) ? 1 : -1);
     return result;
   }
+
+  getRoot(filePath: string): string | undefined {
```

Comment:
I was thinking, should this rather be `merger.realpathSync(filePath);` ? I believe modeling this after `realpathSync` would be ideal, as it is a real and valid fs operation, and also unambiguously will point to the concrete file on disk, regardless of funky links or mergers.

Reply:
`realPathSync` makes more sense. Yes, it will more unambiguous as well.

Reply:
Awesome, I know its some extra work, but I think since this a public API, it's very well worth it.

Reply:
`realPathSync` implementation wasn't the intention here. 
The issue I am trying to solve here is, provided `relative path` or `absolute path`, find the corresponding `root` which is part of the `dirList`.
For Example:
```js
/*
test
 | 
 -- realtest.js
/
 |
 -- user
      |
      -- unit
            |
            -- test.js
*/
let fsMerger = new FSMerger(['fixture/test', '/user/testing/unit']);
let root = fsMerger.getRoot('realtest.js'); // returns `fixture/test`
let fullPath = fsMerger.getRoot('/user/testing/unit/test.js'); // returns `/user/testing/unit`
let relativePath = fsMerger.getRoot('test.js'); // returns `/user/testing/unit`
```